src/x11/screen.c

```diff
@@ -37,31 +37,37 @@ static int x_follow_tear_down_error_handler(void);
 static int FollowXErrorHandler(Display *display, XErrorEvent *e);
 static Window get_focused_window(void);
 
+
+/**
+ * A cache variable to cache the Xft.dpi xrdb values.
+ * We do not expect to change xrdb often, but there's much
+ * overhead to query it once.
+ *
+ * @retval -DBL_MAX: uncached
+ * @retval      <=0: Invalid and unusable value
+ * @retval       >0: valid
+ */
+double screen_dpi_xft_cache = -DBL_MAX;
```

Comment:
`DBL_MAX` should probably be `DBL_MIN`

Reply:
IEEE754 numbers are symmetrical. This works perfectly fine.

I've probably overseen the `DBL_MIN` definition.